dtfs-central-api/src/v1/controllers/tfm/deal/tfm-update-deal.controller.js

```diff
@@ -67,9 +70,14 @@ const updateDeal = async (dealId, dealChanges, existingDeal) => {
 
       dealUpdate.tfm.lastUpdated = new Date().valueOf();
     }
+
+    const updateQuery = produce($.flatten(withoutId(dealUpdate)), (draftQuery) => {
```

Comment:
This might not be needed with new `auditDetails` from https://github.com/UK-Export-Finance/dtfs2/pull/2896